samples/wifi/shell/overlay-scan-only.conf

```diff
@@ -0,0 +1,22 @@
+# Scan only using offload API
+CONFIG_WPA_SUPP=n
+# In 9160 default is socket interface
+CONFIG_NET_DEFAULT_IF_ETHERNET=y
+CONFIG_MBEDTLS=n
+CONFIG_NORDIC_SECURITY_BACKEND=n
+# If MbedTLS is enabled, use the standard files
+CONFIG_GENERATE_MBEDTLS_CFG_FILE=y
+
+# Disable unused networking options
+CONFIG_NET_SOCKETS=n
+CONFIG_POSIX_API=n
+CONFIG_NET_LOG=n
+CONFIG_NET_IPV6=n
+CONFIG_NET_TCP=n
+CONFIG_DNS_RESOLVER=n
+
+# Memories
+CONFIG_NET_PKT_RX_COUNT=1
+CONFIG_NET_PKT_TX_COUNT=1
+# Needed to handle more scan results
+CONFIG_HEAP_MEM_POOL_SIZE=100000
```

Comment:
As mentioned in a chat, this wasn't enough in my trial:

[00:00:00.981,750] <err> wifi_nrf: wifi_nrf_fmac_rx_cmd_send: No space for allocating RX buffer

But got scanning working when increased this to 112640

Reply:
I have missed tweaking the RX buffers, this is now fixed.